Xamarin.Forms.Platform.Android/Material/MaterialProgressBarRenderer.cs

```diff
@@ -13,13 +13,14 @@
 using AProgressBar = Android.Widget.ProgressBar;
 using AView = Android.Views.View;
 
-[assembly: ExportRenderer(typeof(Xamarin.Forms.ProgressBar), typeof(MaterialProgressBarRenderer), new[] { typeof(VisualRendererMarker.Material) })]
+[assembly: ExportRenderer(typeof(ProgressBar), typeof(MaterialProgressBarRenderer), new[] { typeof(VisualRendererMarker.Material) })]
 
 namespace Xamarin.Forms.Platform.Android.Material
 {
 	public class MaterialProgressBarRenderer : AProgressBar,
 		IVisualElementRenderer, IViewRenderer, ITabStop
 	{
+		const float BackgroundAlpha = 0.6f;
```

Comment:
I think this constant should be moved to the `MaterialColors` type (as the rest of the values like this are there)